app/src/main/res/layout/main.xml

```diff
@@ -66,14 +66,14 @@
         android:layout_marginBottom="10dp"
         android:animateLayoutChanges="true"
         android:background="?attr/searchBackgroundColor"
+        android:baselineAligned="false"
         android:clipToPadding="false"
         android:elevation="2dp"
         android:focusableInTouchMode="true"
         android:gravity="center_vertical"
         android:importantForAccessibility="no"
         android:orientation="horizontal"
-        tools:ignore="UnusedAttribute"
-        tools:layout="@layout/favorite_item" />
```

Comment:
Was it required for this PR?

Reply:
No, it could still be there indeed! Thanks